libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/internal/navigation/MapboxNavigationNotification.kt

```diff
@@ -393,14 +396,18 @@ internal class MapboxNavigationNotification : NavigationNotification {
         drivingSide: String?
     ): Boolean {
         return NavigationConstants.STEP_MANEUVER_MODIFIER_LEFT == drivingSide && (
-            NavigationConstants.STEP_MANEUVER_TYPE_ROUNDABOUT == maneuverType ||
-                NavigationConstants.STEP_MANEUVER_TYPE_ROTARY == maneuverType || NavigationConstants.STEP_MANEUVER_MODIFIER_UTURN == maneuverModifier
-            )
+                NavigationConstants.STEP_MANEUVER_TYPE_ROUNDABOUT == maneuverType ||
+                        NavigationConstants.STEP_MANEUVER_TYPE_ROTARY == maneuverType || NavigationConstants.STEP_MANEUVER_MODIFIER_UTURN == maneuverModifier
+                )
     }
 
     private fun onEndNavigationBtnClick() {
-        if (::mapboxNavigation.isInitialized) {
-            mapboxNavigation.stopNavigation()
+        when (::mapboxNavigation.isInitialized) {
```

Comment:
Isn't 

```kotlin
if (::mapboxNavigation.isInitialized) {
  mapboxNavigation.stopNavigation()
}
```

more readable? Note that `else ->` is empty.

Reply:
Yes, in this case you're correct, but in the general case, "when" will fail if you do not provide handlers for each case tested. This makes "when" safer then if-else. You can forget the "else" clause, but with "when", the compiler will remind you.